frontend/src/app/components/incoming-transactions-graph/incoming-transactions-graph.component.ts

```diff
@@ -232,6 +255,7 @@ export class IncomingTransactionsGraphComponent implements OnInit, OnChanges, On
         }
       ],
       yAxis: {
+        max: this.outlierCappingEnabled ? Math.round(this.medianVbytesPerSecond * OUTLIERS_MEDIAN_MULTIPLIER) : undefined,
```

Comment:
```suggestion
        max: this.outlierCappingEnabled ? (value) => Math.min(value.max, this.medianVbytesPerSecond * OUTLIERS_MEDIAN_MULTIPLIER) : undefined,
```

Reply:
not tested, but just an idea to avoid this effect when there aren't any big outliers:


<img width="1256" alt="Screenshot 2023-11-15 at 7 54 59 AM" src="https://github.com/mempool/mempool/assets/83316221/aa96df44-1ef0-4a6b-a4c3-75e013c5a098">
<img width="1256" alt="Screenshot 2023-11-15 at 7 54 51 AM" src="https://github.com/mempool/mempool/assets/83316221/b4594988-8fca-409a-887e-96581be05f39">

Reply:
Implemented this idea a bit differently, nice feedback thanks